tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -952,6 +960,32 @@ callable. This can be used to pass functions as arguments.
           the ``()`` operator directly. This behavior is implemented to avoid
           performance issues on direct function calls.
 
+Lambda functions
+^^^^^^^^^^^^^^^^
+
+Lambda functions allow you to declare functions that do not belong to a class. Instead a Callable object is created and assigned to a variable directly.
+This can be useful to create Callables to pass around without polluting the class scope.
+
+::
+
+    var lambda = func(x): print(x)
+    lambda.call(42) # Prints "42"
+
+Lambdas can be given a name for debugging purposes::
+
+    var lambda = func my_lambda(x):
+        print(x)
+
+They capture the local environment. Local variables are passed by copy, so won't be updated in the lambda if changed in the local function::
```

Comment:
`They` to be explicitly `Lambdas` (or `Lambda functions` if you replace all `Lambdas` with `Lambda functions`)

Passed by value, instead of by copy, anyway final phrase:
```
Lambdas capture the local environment. Local variables are passed by value, so they won't be updated in the lambda if changed in the local function::
```

I haven't heard the term `local enviroment` for any kind of scope, but I think the reader can understand the meaning. Cannot think of something more specific because I am not certain if only current-scope variables are included in `local enviroment` in lambda functions (haven't used them in GDScript)